### PR TITLE
remove the timing on tilings methods and algorithm

### PR DESCRIPTION
**Pull request by enadeau**

### CODE REVIEW

File: tilings/algorithms/gridded_perm_reduction.py [hunk 1]
```diff
@@ -18,7 +16,6 @@ class GriddedPermReduction:
     func_calls = func_calls
     func_times = func_times
```
Comment (christianbean) [inline tilings/algorithms/gridded_perm_reduction.py:17]: We can remove these two dictionaries, as we are not timing anything.

File: tilings/tiling.py [hunk 2]
```diff
@@ -94,7 +93,6 @@ class Tiling(CombinatorialClass):
     func_calls = func_calls
     func_times = func_times
```
Comment (christianbean) [inline tilings/tiling.py:95]: these can go also

Comment (christianbean) [reply tilings/tiling.py]: fixed

File: tilings/algorithms/gridded_perm_reduction.py (hunk 1 above)
Comment (christianbean) [reply tilings/algorithms/gridded_perm_reduction.py]: fixed

**Comment by enadeau** [comment]
Good catch I really forgot to remove the dict. I'll merge

**Pull request merged**